diff --git a/CMakeLists.txt b/CMakeLists.txt
index ecc3f8b..1a4b4db 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -85,7 +85,7 @@ set(CMAKE_CXX_STANDARD 17)
 # don't generally produce global targets. To guarantee all imported targets are 
 # global, this module is included at the project root level.
 
-include(FindExtPackages)
+# include(FindExtPackages)
 
 add_subdirectory(casbin)
 
diff --git a/casbin/CMakeLists.txt b/casbin/CMakeLists.txt
index 17516c0..53ef5ef 100644
--- a/casbin/CMakeLists.txt
+++ b/casbin/CMakeLists.txt
@@ -76,7 +76,7 @@ add_library(casbin STATIC ${CASBIN_SOURCE_FILES})
 
 target_precompile_headers(casbin PRIVATE ${CASBIN_INCLUDE_DIR}/casbin/pch.h)
 target_include_directories(casbin PRIVATE ${CASBIN_INCLUDE_DIR})
-target_link_libraries(casbin PRIVATE nlohmann_json::nlohmann_json)
+# target_link_libraries(casbin PRIVATE nlohmann_json::nlohmann_json)
 
 set_target_properties(casbin PROPERTIES 
     PREFIX ""
